tests/cross-product/sapi/scenarios/add-to-cart/shared-add-to-cart-scenario.js

```diff
@@ -0,0 +1,14 @@
+import { AbstractScenario } from '../../../../abstract-scenario.js';
+import { group } from 'k6';
+
+export class SharedAddToCartScenario extends AbstractScenario {
+    execute() {
+        let self = this;
+        const cartId = self.cartHelper.haveCartWithProducts(0);
+
+        group('Checkout', function () {
+            const params = self.cartHelper.getParamsWithAuthorization();
```

Comment:
I would move this line out of `Checkout` groupping function since this is a preparation step and not related to testing